docs/api-reference/cloud-providers/cloud-provider.md

```diff
@@ -23,9 +23,9 @@ The default provider class
 
 **Parameters**
 
--   `props` **[object][27]** 
-    -   `props.name` **[string][28]** 
-    -   `props.displayName` **[string][28]** 
+-   `props` **[object][27]**
+    -   `props.name` **[string][28]**
```

Comment:
What are those numbers in brackets? Look like link tags - in that case shouldn't those tags be defined below? Or maybe this is a different markdown dialect?

Reply:
the tags are defined at the end of the file https://github.com/keplergl/kepler.gl/blob/d975ea1e043a99039ffedf9771159ba1edfe63b4/docs/api-reference/cloud-providers/cloud-provider.md?plain=1#L209